contributing/beta_components.md

```diff
@@ -0,0 +1,47 @@
+# Beta program for components
```

Comment:
Was this a `git mv` operation that git couldn't track? If it was a copy-paste, can you perform it as a move so we preserve the file history?

Reply:
Looks like git didn't track the rename. Will fix.

Reply:
Git doesn't like the fact that we're changing so much of the file unfortunately.

Reply:
Silly git...